CHANGELOG.md

```diff
@@ -10,6 +11,44 @@ The below sections record the changes for each API version in each release as fo
 * for subsequent release-candidate(s), only the delta to the previous release-candidate
 * for a public release, the consolidated changes since the previous public release
 
+# r1.2
+
+## Release Notes
+This release contains the definition and documentation of 
+* predictive-connectivity-data v0.1.0
+
+The API definition(s) are based on
+* Commonalities v0.6.0
+* Identity and Consent Management v0.4.0
+
+## predictive-connectivity-data v0.1.0
+
+This is a release candidate for the CAMARA Meta Release Fall25 release of the Predictive Connectivity Data API, version v0.1.0. It contains mainly alignments with the Commonalities v0.6.0.
+
+- API definition **with inline documentation**:
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/PredictiveConnectivityData/blob/r1.2/code/API_definitions/predictive-connectivity-data.yaml)
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/PredictiveConnectivityData/r1.2/code/API_definitions/predictive-connectivity-data.yaml&nocors)
+  - [View it on Swagger Editor](https://camaraproject.github.io/swagger-ui/?url=https://raw.githubusercontent.com/camaraproject/PredictiveConnectivityData/r1.2/code/API_definitions/predictive-connectivity-data.yaml)
+
+In the following there is the list of the modifications with respect to the previous release.
+
+### Added
+* N/A
+
+### Changed
+* Alignment with Commonalities r3.3 by @albertoramosmonagas in https://github.com/camaraproject/PredictiveConnectivityData/pull/33
+
+### Fixed
+* N/A
+
+### Removed
+* N/A
+
+ ### New Contributors
+* @eric-murray made their first contribution in https://github.com/camaraproject/PredictiveConnectivityData/pull/30
+
+**Full Changelog**: https://github.com/camaraproject/PredictiveConnectivityData/compare/r1.1...r1.2
```

Comment:
For the first public release, usually all commits that are part of that release are listed
```suggestion
**Full Changelog**: https://github.com/camaraproject/PredictiveConnectivityData/commits/r1.2
```